build.sh

```diff
@@ -60,6 +60,8 @@ echo "==========================================================================
 echo "building..."
 time lerna run $bail --stream $runtarget || fail
 
-/bin/bash scripts/check-api-compatibility.sh
+if [ "${runtarget:-}" != "build" ]; then
+    /bin/bash scripts/check-api-compatibility.sh
+fi
```

Comment:
What's that?

Reply:
If we're going to skip tests, we should skip *all* tests. This made the bump script take ~45 minutes though... only to then get a ~45 minutes PR validation... then a ~45 minutes build... Followed by a ~45 minutes documentation site generation... Who has this kind of time?